pyracf/common/security_admin.py

```diff
@@ -668,6 +650,44 @@ def __format_semi_tabular_data(
                 messages[i + 2][indexes[j] : ind_e1]
             )
 
+    def __format_tabular_data(
```

Comment:
Does this code fix a bug or is this just refactoring?

Reply:
Based on the description for the PR this fixes a problem with profile extract, right?

Reply:
Yes. This change fixed the bug described in the initial issue for this PR and then while I was playing with those settings I added features. The "tabular data" allows the code to grab multiple key-value pairs on one like of text when they use commas to split multiple items under one "header" like with AUDITING.

Reply:
And given that this function is specific to general resource profile, it does not potentially affect anything else.

Reply:
Thank you for the confirmation!